orc/orccpu-arm.c

```diff
@@ -80,10 +80,18 @@ orc_check_neon_proc_auxv (void)
     }
 
     if (aux[0] == AT_HWCAP) {
+      #ifdef __arm__
```

Comment:
Indent as other codes do for #if/#endif.

Reply:
Got it